how-to/get-started/quick-start-with-tidb.md

```diff
@@ -0,0 +1,316 @@
+
+## 第二种：使用 TiUP Playground 快速部署本地测试环境
+
+- 适用场景：利用本地 Mac 或者单机 Linux 环境快速部署 TiDB 集群。可以体验 TiDB 集群的基本架构，以及 TiDB、TiKV、PD、监控等基础组件的运行。
+- 耗时：1 分钟
+
+作为一个分布式系统，最基础的 TiDB 测试集群通常由 2 个 TiDB 实例、3 个 TiKV 实例和 3 个 PD 实例来构成。通过 TiUP Playground，可以快速搭建出上述的一套基础测试集群。
+
+1. 下载并安装 TiUP。
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+    ```
+
+2. 声明全局环境变量。
+
+    > **注意：**
+    >
+    > TiUP 安装完成会提示对应的 profile 文件的绝对路径，以下 source 操作需要根据实际位置进行操作。
+
+    {< copyable "shell-regular" >}}
+
+    ```shell
+    source .bash_profile
+    ``` 
+
+3. 安装 TiUP Playground。
+  
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup install playground
+    ```
+  
+    如果曾经安装过 TiUP Playground，则执行以下命令升级至最新版：
+  
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup update playground
+    ```
+
+4. 在当前 session 执行以下命令启动集群。
+
+    - 直接运行 `tiup playground` 命令会运行最新版本的 TiDB 集群，其中 TiDB、TiKV 和 PD 实例各 1 个：
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground
+        ```
+
+    - 也可以指定版本以及组件实例个数：
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground v4.0.0-rc --db 2 --pd 3 --kv 3 --monitor
+        ```
+
+        上述命令会在本地下载并启动一个 v4.0.0-rc 版本的集群，`--monitor` 表示同时部署监控组件。运行结果将显示集群的访问方式：
+        
+        ```log
+        CLUSTER START SUCCESSFULLY, Enjoy it ^-^
+        To connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
+        To connect TiDB: mysql --host 127.0.0.1 --port 4001 -u root
+        To view the dashboard: http://127.0.0.1:2379/dashboard
+        To view the monitor: http://127.0.0.1:9090
+        ```
+
+5. 新开启一个 session 以访问 TiDB 数据库。
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    mysql --host 127.0.0.1 --port 4000 -u root
+    ```
+
+6. 通过 <http://127.0.0.1:9090> 访问 TiDB 的 Prometheus 管理界面。
+
+7. 通过 <http://127.0.0.1:2379/dashboard> 访问 TiDB 的 Dashboard 页面，默认用户名为 root，密码为空。
+
+8. 测试完成后清理集群，绿色环保。通过 `ctrl-c` 停掉进程后，执行以下命令：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup clean --all
+    ```
+
+## 第三种：使用 TiUP cluster 模拟单机下的准生产测试环境
+
+- 适用场景：希望用单台 Linux 服务器，体验 TiDB 最小的完整拓扑的集群，并模拟生产的部署步骤。
+- 耗时：10 分钟
+
+本节介绍如何参照 TiUP 最小拓扑的一个 YAML 文件部署 TiDB 集群。
+
+### 准备环境
+
+准备一台部署主机，确保其软件满足需求：
+
+- 推荐安装 CentOS 7.3 及以上版本
+- Linux 操作系统开放外网访问，用于下载 TiDB 及相关软件安装包
+
+最小规模的 TiDB 集群拓扑：
+
+| 实例 | 个数 | IP | 配置 |
+|:-- | :-- | :-- | :-- |
+| TiKV | 3 | 10.0.1.1 <br> 10.0.1.1 <br> 10.0.1.1 | 避免端口和目录冲突 |
+| TiDB | 1 | 10.0.1.1 | 默认端口 <br> 全局目录配置 |
+| PD | 1 | 10.0.1.1 | 默认端口 <br> 全局目录配置 |
+| TiFlash | 1 | 10.0.1.1 | 默认端口 <br> 全局目录配置 |
+| Monitor | 1 | 10.0.1.1 | 默认端口 <br> 全局目录配置 |
+
+部署主机软件和环境要求：
+
+- 部署需要使用部署主机的 root 用户及密码
+- 部署主机[关闭防火墙](/how-to/deploy/orchestrated/tiup.md#如何关闭部署机器的防火墙)或者开放 TiDB 集群的节点间所需端口
+- 目前 TiUP 仅支持在 x86_64 (AMD64) 架构上部署 TiDB 集群（TiUP 将在 4.0 GA 时支持在 ARM 架构上部署）
+    - 在 AMD64 架构下，建议使用 CentOS 7.3 及以上版本 Linux 操作系统
+    - 在 ARM 架构下，建议使用 CentOS 7.6 1810 版本 Linux 操作系操作步骤：
```

Comment:
typo

Reply:
fixed in https://github.com/pingcap/docs-cn/pull/2813